splunk-otel-android/src/main/java/com/splunk/rum/FileUtils.java

```diff
@@ -0,0 +1,58 @@
+package com.splunk.rum;
+
+import static com.splunk.rum.SplunkRum.LOG_TAG;
+
+import android.util.Log;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+// Basic wrapper around filesystem operations, primarily for testing
+public class FileUtils {
```

Comment:
This probably does not need to be a part of public API
```suggestion
final class FileUtils {
```